ecs-cli/modules/utils/compose/convert_task_definition_test.go

```diff
@@ -114,6 +115,71 @@ func TestConvertToTaskDefinition(t *testing.T) {
 	assert.Equal(t, taskRoleArn, aws.StringValue(taskDefinition.TaskRoleArn), "Expected taskRoleArn to match")
 }
 
+func TestConvertToTaskDefinitionWithECSFields(t *testing.T) {
+
+	ecsFieldsString := `version: 1
+task_definition:
+  ecs_network_mode: host
+  task_role_arn: arn:aws:iam::123456789012:role/my_role`
+
+	content := []byte(ecsFieldsString)
+	tmpfile, err := ioutil.TempFile("", "ecs-fields")
+
+	if err != nil {
+		t.Errorf("Could not create ecs fields tempfile due to error: %v", err)
+	}
+
+	defer os.Remove(tmpfile.Name())
+
+	if _, err := tmpfile.Write(content); err != nil {
+		t.Errorf("Could not write data to ecs fields tempfile due to error: %v", err)
+	}
+
+	if err := tmpfile.Close(); err != nil {
+		t.Errorf("Could not close tempfile due to error: %v", err)
+	}
+
+	ecsFieldsFileName := tmpfile.Name()
+
+	taskDefinition := convertToTaskDefWithEcsFieldsInTest(t, "name", &config.ServiceConfig{}, "", ecsFieldsFileName)
+
+	assert.Equal(t, "host", aws.StringValue(taskDefinition.NetworkMode), "Expected network mode to match")
+	assert.Equal(t, "arn:aws:iam::123456789012:role/my_role", aws.StringValue(taskDefinition.TaskRoleArn), "Expected task role arn to match")
```

Comment:
Can we use 'ARN' here in the error message output?

Reply:
done.